sdk/cmake/AddPlatformLibrary.cmake

```diff
@@ -133,6 +133,8 @@ function(ADD_PLATFORM_LIBRARY NAME)
         )
     else()
         add_library(${NAME} SHARED ${SOURCES})
+        find_package(OpenCV)
+        target_link_libraries(${NAME} PRIVATE ${OpenCV_LIBS})
```

Comment:
this branch only handles non-web platforms
also most likely the right place for this code is in `CMakeLists.txt` file instead of here, and i think you could use `link_platform_libraries()` instead of `target_link_libraries()` with `HIDE_SYMBOLS` option

the reason why this code works without you appending the path to `CMAKE_MODULE_PATH ` is that it is finding the OS-installed opencv instead of the submodule